rs/nns/governance/canister/governance.did

```diff
@@ -577,6 +626,62 @@ type Neuron = record {
   transfer : opt NeuronStakeTransfer;
   known_neuron_data : opt KnownNeuronData;
   spawn_at_timestamp_seconds : opt nat64;
+
+  // When a neuron is first created, this is set to create_timestamp_seconds.
+  //
+  // This is part of the "periodic confirmation" feature, which was proposed and
+  // (overwhelmingly) adopted in NNS motion proposal 132411:
+  // https://dashboard.internetcomputer.org/proposal/132411
+  //
+  // This gets updated to the current time when the user performs related
+  // actions, as described in the aforementioned proposal.
+  //
+  // For neurons created prior to this feature, this field will be set to
+  // 2024-09-01T00:00:00 UTC.
+  //
+  // Per NNS policy, this is opt. Nevertheless, it will never be null (unless we
+  // later deprecate this feature).
+  voting_power_refreshed_timestamp_seconds : opt nat64;
+
+  // The amount of "sway" this neuron has when voting on proposals.
+  //
+  // If a neuron regularly refreshes its voting power, this value is the same as
+  // potential_voting_power.
+  //
+  // However, if a neuron has not refreshed in more than six months, this will
+  // be less than potential voting power.
+  //
+  // As a further secondary consequence, the neuron receives less voting rewards
+  // (when it votes indirectly via following).
+  //
+  // For details, see https://dashboard.internetcomputer.org/proposal/132411.
+  //
+  // Per NNS policy, this is opt. Nevertheless, it will never be null (unless we
+  // later deprecate this feature).
+  deciding_voting_power : opt nat64;
+
+  // The amount of "sway" this neuron has when voting on proposals, assuming
+  // that it has refreshed recently enough.
+  //
+  // Unlike deciding_voting_power, this does NOT take refreshing into account.
+  // Rather, this only takes three factors into account:
+  //
+  //     1. (Net) staked amount - This is the "base" of a neuron's voting power.
+  //        This primarily consists of the balance in the neuron's NNS
+  //        Governance subaccount, but also includes (unburned) fees, and staked
```

Comment:
I was a bit confused by "this includes fees" - whether this means that the fees are accounted for or not. We could also just remove the second phrase as this should probably also be explained in the introduction of the stake field

Reply:
You mean, delete the part starting from ", but also"? Sure.

Reply:
yes exactly

Reply:
Done.